server/fishtest/rundb.py

```diff
@@ -861,6 +861,10 @@ def priority(run):  # lower is better
         for task in run["tasks"]:
             opening_offset += task["num_games"]
 
+        if "sprt" in run:
```

Comment:
```python
if "sprt" in run["args"]:
    sprt_batch_size_games = 2 * run["args"]["sprt"]["batch_size"]
    remaining = sprt_batch_size_games * math.ceil(remaining / sprt_batch_size_games)

```

Reply:
Thanks! Sorry about that.... I will fix it.

Reply:
Done...